src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -91,6 +102,29 @@ public void setPerson(Person target, Person editedPerson) {
      */
     public void removePerson(Person key) {
         persons.remove(key);
+        tags.removeTags(key.getTags());
+    }
+
+    //// tag-level methods
```

Comment:
Can delete one set of //

Reply:
It is the default one, if you see on top and below.